test/index.js

```diff
@@ -4,6 +4,7 @@ const { expect } = require('chai');
 const rp = require('request-promise');
 const enableDestroy = require('server-destroy');
 const apostropheMultisite = require('./app.js');
+const Promise = require('bluebird');
```

Comment:
It doesn't looks like `Promise` is used in this file.

Reply:
Yeah, cleaned up (and we now pass eslint, and eslint is now run by npm test).